api/nodes/serializers.py

```diff
@@ -181,7 +163,7 @@ def update(self, instance, validated_data):
 
 class NodeFilesSerializer(JSONAPISerializer):
 
-    id = ser.CharField(read_only=True, source='_id')
+    id = ser.SerializerMethodField()
```

Comment:
jsonapi requires an `id` and `type` under `data`.  If `id` was a CharField here, it was returning the following `id` for a folder:

![screen shot 2015-08-28 at 4 05 16 pm](https://cloud.githubusercontent.com/assets/9755598/9555873/55c480a6-4d9e-11e5-8593-f930ba3d2847.png)

I was told to make it a mix of `provider` and `path`.  There isn't really an `id` here (list of all Node files).  I think this is a good substitution.